e2etest/bgptests/bgp_multiprotocol.go

```diff
@@ -106,23 +106,23 @@ var _ = ginkgo.Describe("BGP Multiprotocol", func() {
 			ginkgo.Entry("DUALSTACK - via ipv4",
 				ipfamily.IPv4, []string{v4PoolAddresses, v6PoolAddresses}, func(svc *corev1.Service) {
 					testservice.TrafficPolicyCluster(svc)
-					testservice.DualStack(svc)
+					testservice.RequiredDualStack(svc)
```

Comment:
just wondering, why you didn't change teh entry name here (and I think we should discuss if it makes sense or not).